docs/guides/transpile-rest-api.mdx

```diff
@@ -15,17 +15,17 @@ You have two transpilation options:
 
 * Transpile [locally via Qiskit](./defaults-and-configuration-options) before generating the QASM string.
 
-* Use the [Qiskit Transpiler Service Python client](/api/qiskit-ibm-transpiler) or [Qiskit Transpiler Service REST API](/api/qiskit-transpiler-service-rest).
+* Use the [Qiskit Transpiler Service Python client](/docs/api/qiskit-ibm-transpiler) or [Qiskit Transpiler Service REST API](/docs/api/qiskit-transpiler-service-rest).
 
 <Admonition type="note">
-    [Transpilation is necessary](/announcements/product-updates/2024-02-14-qiskit-runtime-primitives-update) prior to submitting a circuit to IBM&reg; QPUs.
+    [Transpilation is necessary](/docs/announcements/product-updates/2024-02-14-qiskit-runtime-primitives-update) prior to submitting a circuit to IBM&reg; QPUs.
```

Comment:
I realized this will break on the cloud app. I opened https://github.com/Qiskit/documentation/issues/3025